### PR TITLE
Add test for CA with CRL publishing

### PR DESCRIPTION
**Pull request by edewata**
Docs:

* https://github.com/dogtagpki/pki/wiki/Configuring-File-based-CRL-Publishing

### CODE REVIEW

**Review by ckelleyRH — APPROVED** [review]
LGTM

File: .github/workflows/ca-tests.yml [hunk 1]
```diff
@@ -1838,6 +1838,217 @@ jobs:
           path: |
             /tmp/artifacts/pki
 
+  ca-crl-test:
+    name: Testing CA with CRL publishing
```
Comment (ladycfu) [inline .github/workflows/ca-tests.yml:1842]: Might want to stress that this is only "file-based" CRL publishing

**Review by ladycfu — APPROVED** [review]
only one simple suggestion in the comment for test clarity.

**Comment by edewata** [comment]
@ckelleyRH @ladycfu I updated the PR as suggested. Thanks!

**Pull request merged**